cypress/features/regression/test/actionPopoverIFrameNoIFrame.feature

```diff
@@ -47,6 +47,18 @@ Feature: Action Popover component
     Then focused element inner content is set to "Business"
```

Comment:
in this scenario missing step which changes the default focus from the Business element.